Detections/CommonSecurityLog/MultiVendor-PossibleDGAContacts.yaml

```diff
@@ -111,4 +111,9 @@ query: |
     )
     on Name
     | project StartTime, EndTime, Name, DGADomain, SourceIP, DestinationIP, DataSource
-    | extend timestamp=StartTime, IPCustomEntity=SourceIP
\ No newline at end of file
+    | extend timestamp=StartTime, IPCustomEntity=SourceIP
+entityMappings:
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: IPCustomEntity
```

Comment:
Are we ready to add in the DNS.DomainName entity yet?  It would be great if we could include this -
`| extend DNSCustomEntity = Name`
  \- entityType: DNS
    fieldMappings:
         - identifier: DomainName
       columnName: DNSCustomEntity

Reply:
great point @shainw 
Sure, this should be available in the new mapping model. However, it won't work in the old one, but it is totally fine. we will be moving to the new mapping model slowly until we are fully using it. For now, users with the new model would benefit from it, and it won't have an effect on the old one, so feel free to use it. More details about all the supported entity types in the mail I sent.

In the meantime, since we will live with both mappings for a while, we would still need to maintain both mapping models. So we still want to have the **CustomEntity in the query. Exactly as you did above. Once we are only working with the new mapping model (it will take a while), we can use any column names we want for the entity fields :)